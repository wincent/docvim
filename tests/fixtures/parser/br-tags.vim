""
"
" This.<br>
" That.
"
" This.<br />
" That.
"
" This.<br/>
" That.
"
" This.<br>That.
"
" Note that surrounding whitespace    <br>     
" gets slurped (at the end of the line).
"
" And      <br />    internally too.
"
" Also,<BR>
" we<Br>
" are<bR>
" case<Br />
" insensitive.<BR />
"
