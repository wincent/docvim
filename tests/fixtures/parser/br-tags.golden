Project
  [ Project
      [ Unit
          [ DocBlock
              [ Paragraph [ Plaintext "This." , BreakTag , Plaintext "That." ]
              , Paragraph [ Plaintext "This." , BreakTag , Plaintext "That." ]
              , Paragraph [ Plaintext "This." , BreakTag , Plaintext "That." ]
              , Paragraph [ Plaintext "This." , BreakTag , Plaintext "That." ]
              , Paragraph
                  [ Plaintext "Note"
                  , Whitespace
                  , Plaintext "that"
                  , Whitespace
                  , Plaintext "surrounding"
                  , Whitespace
                  , Plaintext "whitespace"
                  , BreakTag
                  , Plaintext "gets"
                  , Whitespace
                  , Plaintext "slurped"
                  , Whitespace
                  , Plaintext "(at"
                  , Whitespace
                  , Plaintext "the"
                  , Whitespace
                  , Plaintext "end"
                  , Whitespace
                  , Plaintext "of"
                  , Whitespace
                  , Plaintext "the"
                  , Whitespace
                  , Plaintext "line)."
                  ]
              , Paragraph
                  [ Plaintext "And"
                  , BreakTag
                  , Plaintext "internally"
                  , Whitespace
                  , Plaintext "too."
                  ]
              , Paragraph
                  [ Plaintext "Also,"
                  , BreakTag
                  , Plaintext "we"
                  , BreakTag
                  , Plaintext "are"
                  , BreakTag
                  , Plaintext "case"
                  , BreakTag
                  , Plaintext "insensitive."
                  , BreakTag
                  ]
              ]
          ]
      ]
  ]
